src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -420,10 +420,12 @@ Possible matches are</value>
     <value>Missing statement block in switch statement clause.</value>
   </data>
   <data name="MissingForeachExpression" xml:space="preserve">
-    <value>Missing expression in foreach loop.</value>
+    <value>Missing expression in foreach loop.
+The correct form is: foreach ($a in $b) {}</value>
```

Comment:
Would this be slightly more informative?

```none
The correct form is: foreach ($a in $b) { ... }
```

Reply:
Good Q.
I decided to ask users:
https://twitter.com/jsnover/status/914622511412891649

Reply:
Good call!
Twitter is pretty clear on this:
![image](https://user-images.githubusercontent.com/6188114/31060954-6cccb36c-a6d0-11e7-96fb-1fe790d5615e.png)